src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ITypeOfExpression.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
```

Comment:
I believe the BOM is expected and was inadvertently removed previously due to this issue: https://github.com/dotnet/roslyn/issues/22319